spec/integration/prometheus_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative '../vcr/vcr_setup'
+require_relative '../spec_helper'
+
+module Hawkular::Prometheus::RSpec
+  HAWKULAR_BASE = 'http://localhost:8080/'
+  creds = {}
+  options = {}
+
+  describe 'Prometheus/Queries' do
+    let(:cassette_name) do |example|
+      description = example.description
+      description
+    end
+
+    around(:each) do |example|
+      record('Prometheus/Queries', credentials, cassette_name, example: example)
+    end
+
+    before(:each) do
+      @client = Hawkular::Prometheus::Client.new(HAWKULAR_BASE, creds, options)
```

Comment:
Use a `let` definition instead of defining instance variables.

Reply:
+1